src/aragonjs-wrapper.js

```diff
@@ -64,6 +64,25 @@ const appBaseUrl = (app, gateway = ipfsDefaultConf.gateway) => {
 const applyAppOverrides = apps =>
   apps.map(app => ({ ...app, ...(appOverrides[app.appId] || {}) }))
 
+const hasWebApp = app => Boolean(app['start_url'])
+
+const getAPMRegistry = ({ appName = '' }) =>
+  appName.substr(appName.indexOf('.') + 1) // everything after the first '.'
+
+const getAppTags = app => {
```

Comment:
Suggestion:

```JavaScript
const { status=[] } = app
return [
  ...(status ? [status] : []),
  ...(apmRegistry !== 'aragonpm.eth' ? [${apmRegistry} registry`] : []),
  ...(!hasWebApp(app) ? ['contract-only'] : []),
]
```

Reply:
Opted to use `.push()` instead; although more verbose, I think it's a lot clearer.